api/config_entry.go

```diff
@@ -196,6 +196,16 @@ type PassiveHealthCheck struct {
 	// MaxFailures is the count of consecutive failures that results in a host
 	// being removed from the pool.
 	MaxFailures uint32 `alias:"max_failures"`
+
+	// EnforcingConsecutive5xx is the % chance that a host will be actually ejected
+	// when an outlier status is detected through consecutive 5xx.
+	// This setting can be used to disable ejection or to ramp it up slowly. Defaults to 100.
```

Comment:
I'm not sure we are setting the default value, so this will result to not doing any ejection, right? 
We should either change this doc or change the default value.